packages/office-ui-fabric-react/src/components/SearchBox/examples/SearchBox.FullSize.Example.tsx

```diff
@@ -1,25 +1,17 @@
 import * as React from 'react';
 import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
-import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { Stack, IStackTokens } from 'office-ui-fabric-react/lib/Stack';
 
-// tslint:disable:jsx-no-lambda
+const stackTokens: Partial<IStackTokens> = { childrenGap: 20 };
 
+// tslint:disable:jsx-no-lambda
 export const SearchBoxFullSizeExample = () => {
   return (
-    <Stack tokens={{ childrenGap: 20 }}>
-      <SearchBox
-        placeholder="Search"
-        onSearch={newValue => console.log('value is ' + newValue)}
-        onFocus={() => console.log('onFocus called')}
-        onBlur={() => console.log('onBlur called')}
-        onChange={() => console.log('onChange called')}
-      />
+    <Stack tokens={stackTokens}>
+      <SearchBox placeholder="Search" onSearch={newValue => console.log('value is ' + newValue)} />
```

Comment:
This should go in a constant at the top of the file too (and then you can remove the tslint-disable):
`const onSearch = newValue => console.log('value is ' + newValue)`